request.go

```diff
@@ -247,3 +247,25 @@ type RefundReq struct {
 	Amount    int64  `json:"amount"`
 	Reason    string `json:"reason"`
 }
+
+// Subscribe : Represent Subscribe object payload (request and response)
+type Subscribe struct {
```

Comment:
Can we name it `SubscribeReq` to make it more consistent with the rest of struct?

Reply:
cause it also used as repsonse. Wdyt?

Reply:
let's discuss on https://github.com/veritrans/go-midtrans/pull/55#discussion_r361421651